x-pack/test_serverless/api_integration/services/svl_cases/api.ts

```diff
@@ -133,6 +133,7 @@ export function SvlCasesApiServiceProvider({ getService }: FtrProviderContext) {
         status: CaseStatuses.open,
         updated_by: null,
         category: null,
+        customFields: [],
```

Comment:
Same comment as before about custom fields in the post request.

Reply:
Kept  `customFields: []`  because it's in `postCaseResp`

Reply:
My bad!!